uint8 ptpMode
float32 x
float32 y
float32 z
float32 r
---
int32 result
uint64 queuedCmdIndex
